### PR TITLE
Side margin

### PR DESCRIPTION
**Pull request by prem43**
Fullscreen and Exitscreen with Sidemargin

### CODE REVIEW

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/appsettings.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
 
   "ConnectionStrings": {
-    "IdentityDBLocal": "Server=localhost;Database=TestingAndCalibrationLabs; User ID=sa;Password=admin;"
+    "IdentityDBLocal": "Server=LENOVO;Database=SideMargin; User ID=sa;Password=admin;"
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/appsettings.json:12]: dont checkin appsettings.json file until there is something specific reason or updating any other key

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml [hunk 2]
```diff
@@ -98,4 +112,24 @@
...
+            Element.style = "padding-right: 0px;padding-left: 0px;";
+            document.getElementById('nav-item-FullPage').innerHTML = "Exit FullPage";
+        }
+        else {
+            var View = window.matchMedia("(min-width: 600px)")
+            Page = 0;
+            if (View.matches) {
+                Element.style = "padding-right:230px;padding-left: 230px;";
+            }
+            document.getElementById('nav-item-FullPage').innerHTML = "FullPage";
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:132]: if i click 5-10 time continuously then style will only update or will add again and again style ex :-  "padding-right:230px;padding-left: 230px;"  "padding-right:230px;padding-left: 230px;" "padding-right:230px;padding-left: 230px;" "padding-right:230px;padding-left: 230px;"

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml [hunk 3]
```diff
@@ -63,16 +67,17 @@
...
                     }
-                    else if(nameExistOrNot) {
+                    else if (nameExistOrNot) {
                         nameExistOrNot.style.display = "block";
                     }
                     var divExistOrNot = document.getElementById(parentListId);//Its not being used anywhere who to take
                     response[i].childrens.sort(function (a, b) {
+
                         if (a.name < b.name) { return -1; }
                         if (a.name > b.name) { return 1; }
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:77]: remove extra spaced

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml [hunk 4]
```diff
@@ -4,6 +4,10 @@
             <i class="fa fa-gear"></i>
         </a>
         <div class="dropdown-menu dropdown-menu-right" id="dropdownContentSettings" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" style="display:none" id="nav-item-FullPage" onclick="toggleFullScreen(++Page)">
+                FullPage
+
+            </a>
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:10]: extra space remove

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -46,9 +46,25 @@
...
             display: none;
         }
+
+        
+        @@media screen and (max-width: 598px) {
+            #PageBody {
+                padding-right: 0px;
+                padding-left: 0px;
+            }
+        }
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_Layout.cshtml:67]: extra space

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml (hunk 2 above)
Comment (rajguptaH) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:132]: what if i click again and again in Full page and exit full page then what is going to happend

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml [hunk 6]
```diff
@@ -98,4 +112,24 @@
             }
         });
     });
-</script>
\ No newline at end of file
+</script>
+<script>
```
Comment (rajguptaH) [inline TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:116]: use single script tag you have created multi script tags

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/appsettings.json (hunk 1 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/appsettings.json:12]: ok

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml (hunk 6 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:116]: i have  remove it.

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_Layout.cshtml (hunk 5 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_Layout.cshtml:67]: i have remove it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml (hunk 4 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:10]: i have remove it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml (hunk 3 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:77]: i have remove it

File: TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml (hunk 2 above)
Comment (prem43) [reply TestingAndCalibrationLabs/TestingAndCalibrationLabs.Web.UI/Views/Shared/_pageNavigation.cshtml:132]: If you click on full screen and exit screen again and again, then as many times you will go full screen, then there will be a button of exit screen, which you will do first exit full screen, only then you will go full screen.

**Pull request merged**